onadata/libs/utils/cache_tools.py

```diff
@@ -41,3 +44,11 @@ def safe_delete(key):
 def safe_key(key):
     """Return a hashed key."""
     return hashlib.sha256(force_bytes(key)).hexdigest()
+
+def cache_single_object(key_prefix, id, query):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'id'](https://app.codacy.com/manual/Ona/onadata/pullRequest?prid=5478081)